tools/spellcheck/correct_words.txt

```diff
@@ -763,6 +784,12 @@ ltnote
 ###################################
 # common technical words {{{
 ###################################
+FHWM
+datasets
+Fortran
+NETLIB
+nonrobust
+rubstifying
```

Comment:
In the link you provided it says
`The number of ‘robustifying’ iterations ...`

... which makes more sense actually :)

Am 11/19/2015 um 4:56 PM schrieb Hannes Roest:
> In tools/spellcheck/correct_words.txt
> <https://github.com/OpenMS/OpenMS/pull/1710#discussion_r45358554>:
>
>> @@ -763,6 +784,12 @@ ltnote
>>  ###################################
>>  # common technical words {{{
>>  ###################################
>> +FHWM
>> +datasets
>> +Fortran
>> +NETLIB
>> +nonrobust
>> +rubstifying
>
> hmm I think it is used in the comments of the original code and is also
> in the R documentation
> http://stat.ethz.ch/R-manual/R-patched/library/stats/html/lowess.html
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenMS/OpenMS/pull/1710/files#r45358554>.
>